src/SprykerEco/Yves/Payone/Form/DataProvider/BancontactOnlineTransferDataProvider.php

```diff
@@ -48,6 +48,6 @@ public function getOptions(AbstractTransfer $quoteTransfer): AbstractTransfer
      */
     protected function getBankCountries(): array
     {
-        return PayoneConstants::PAYONE_BANCONTACT_AVAILABLE_COUNTRIES;
+        return PayoneConfig::PAYONE_BANCONTACT_AVAILABLE_COUNTRIES;
```

Comment:
Now you bound to the exact config class, what makes it impossible to extend on project level.
Please create a method in the config and use resolvable config from the factory to access it.

Reply:
This PAYONE_BANCONTACT_AVAILABLE_COUNTRIES config is immutable since this config defines countries for specific payment method. It's fine to have it like this.

Reply:
But as I can see by usage you expect that projects change it. So come over/let me know if you didn't get the issue.